src/include/reconcile_inline.h

```diff
@@ -92,108 +92,75 @@ __rec_page_time_stats(WT_SESSION_IMPL *session, WT_RECONCILE *r)
         r->rec_page_cell_with_txn_id = true;
     }
     if (r->count_durable_stop_ts != 0) {
-        WT_STAT_CONN_INCRV(
-          session, rec_time_window_bytes_ts, r->count_durable_stop_ts * sizeof(wt_timestamp_t));
-        WT_STAT_DATA_INCRV(
+        WT_STAT_CONN_DATA_INCRV(
           session, rec_time_window_bytes_ts, r->count_durable_stop_ts * sizeof(wt_timestamp_t));
-        WT_STAT_CONN_INCRV(session, rec_time_window_durable_stop_ts, r->count_durable_stop_ts);
-        WT_STAT_DATA_INCRV(session, rec_time_window_durable_stop_ts, r->count_durable_stop_ts);
-        WT_STAT_CONN_INCR(session, rec_time_window_pages_durable_stop_ts);
-        WT_STAT_DATA_INCR(session, rec_time_window_pages_durable_stop_ts);
+        WT_STAT_CONN_DATA_INCRV(session, rec_time_window_durable_stop_ts, r->count_durable_stop_ts);
+        WT_STAT_CONN_DATA_INCR(session, rec_time_window_pages_durable_stop_ts);
         r->rec_page_cell_with_ts = true;
     }
     if (r->count_stop_ts != 0) {
-        WT_STAT_CONN_INCRV(
+        WT_STAT_CONN_DATA_INCRV(
           session, rec_time_window_bytes_ts, r->count_stop_ts * sizeof(wt_timestamp_t));
-        WT_STAT_DATA_INCRV(
-          session, rec_time_window_bytes_ts, r->count_stop_ts * sizeof(wt_timestamp_t));
-        WT_STAT_CONN_INCRV(session, rec_time_window_stop_ts, r->count_stop_ts);
-        WT_STAT_DATA_INCRV(session, rec_time_window_stop_ts, r->count_stop_ts);
-        WT_STAT_CONN_INCR(session, rec_time_window_pages_stop_ts);
-        WT_STAT_DATA_INCR(session, rec_time_window_pages_stop_ts);
+        WT_STAT_CONN_DATA_INCRV(session, rec_time_window_stop_ts, r->count_stop_ts);
+        WT_STAT_CONN_DATA_INCR(session, rec_time_window_pages_stop_ts);
         r->rec_page_cell_with_ts = true;
     }
     if (r->count_stop_txn != 0) {
-        WT_STAT_CONN_INCRV(
-          session, rec_time_window_bytes_txn, r->count_stop_txn * sizeof(uint64_t));
-        WT_STAT_DATA_INCRV(
+        WT_STAT_CONN_DATA_INCRV(
           session, rec_time_window_bytes_txn, r->count_stop_txn * sizeof(uint64_t));
-        WT_STAT_CONN_INCRV(session, rec_time_window_stop_txn, r->count_stop_txn);
-        WT_STAT_DATA_INCRV(session, rec_time_window_stop_txn, r->count_stop_txn);
-        WT_STAT_CONN_INCR(session, rec_time_window_pages_stop_txn);
-        WT_STAT_DATA_INCR(session, rec_time_window_pages_stop_txn);
+        WT_STAT_CONN_DATA_INCRV(session, rec_time_window_stop_txn, r->count_stop_txn);
+        WT_STAT_CONN_DATA_INCR(session, rec_time_window_pages_stop_txn);
         r->rec_page_cell_with_txn_id = true;
     }
 
-    if (r->count_prepare != 0) {
-        WT_STAT_CONN_INCRV(session, rec_time_window_prepared, r->count_prepare);
-        WT_STAT_DATA_INCRV(session, rec_time_window_prepared, r->count_prepare);
-        WT_STAT_CONN_INCR(session, rec_time_window_pages_prepared);
-        WT_STAT_DATA_INCR(session, rec_time_window_pages_prepared);
+    if (r->count_prepare != 0)
```

Comment:
I don't think removing the braces here is right. Only the first statement gets executed.